curriculum/challenges/english/14-responsive-web-design-22/learn-responsive-web-design-by-building-a-piano/612eaf56b7ba3257fdbfb0db.md

```diff
@@ -27,7 +27,7 @@ assert(img?.nextElementSibling?.className === 'keys');
 assert(img?.previousElementSibling === null);
 ```
 
-Your `img` element should have a `class` set to logo.
+Your `img` element should have a `class` set to `logo`.
```

Comment:
please remove this change from your PR

Reply:
Sorry this was an earlier pull request that was approved. I'm kind of new to github and i'm not sure why it is showing here.

Reply:
How would I remove the unrelated commit?

Reply:
There's a couple of ways to do that. The simplest might be just reverting changes manually and adding new commit with those reverts. This adds another commit, but when merging all changes will be squashed into a single commit anyway.

Some alternative is using a so-called interactive rebase, to completely remove any traces of not needed commits. However as this approach changes history of the branch, after the rebase it will be needed to `--force` push branch to your fork.

Later make sure your `main` branch locally is in sync with the freeCodeCamp's `main` branch. This is likely how the unrelated commit sneaked to this PR. Follow the step `2` from https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally